src/Options.ts

```diff
@@ -13,6 +13,7 @@ function getDefaultOptions<I, O, E>(): TestFolderOptions<I, O, E> {
     return {
         assertOnError,
         assertOnResult,
+        checkForExcessKeys: true,
```

Comment:
Now that I think of it, _this_ is the backwards incompatible change. Their tests that used to pass now won't even run on the bot